pkg/opencontainer/specs.go

```diff
@@ -0,0 +1,437 @@
+package opencontainer
+
+import (
+	"os"
+)
+
+// These struct definitions are copied verbatim from "github.com/opencontainers/specs".
+// The original source package includes Linux-specific structures configured to be
+// compiled only on Linux systems, making it unnecessarily difficult to develop P2 from
+// other environments.
```

Comment:
Can you enumerate what they were? I'm slightly worried about maintaining this file.